docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -21,60 +23,116 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Enabling GitHub pull request comments
+# Set up GitHub pull request comments
+
+<!--  The entire process of setting up the GH comment is more than just "enabling it", ie. turning it on. Users have to set up the rules. So I changed the verb. -->
+
+Semgrep can create **pull request (PR) comments** in your GitHub repository. These comments inform your developers of **findings**, such as security issues, in their PRs. Semgrep can also provide remediation tips or code fixes that your developers can click to **commit** into their code directly.
 
 :::info Prerequisites
-* Pull request (PR) comments can only be enabled through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
-* To receive alerts and notifications, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+- Pull request (PR) comments can only be set up through Semgrep Cloud Platform (SCP). [<i class="fas fa-external-link fa-xs"></i> Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up PR comments.
+- You must connect your GitHub organization (org) to Semgrep.
+    - For **GitHub Cloud**: See [<i class="fa-regular fa-file-lines"></i> Adding an organization](/semgrep-cloud-platform/getting-started/#adding-an-organization).
+    - For **GitHub Enterprise Server**: [<i class="fa-regular fa-file-lines"></i> Integrate Semgrep with GitHub Enterprise](/semgrep-cloud-platform/scm/#integrating-semgrep-cloud-platform-with-github-enterprise-or-gitlab-self-managed) and if you have a VPN or firewall, [set up your allowlists](/semgrep-cloud-platform/scm/#receiving-pr-or-mr-comments-in-your-vpn-or-on-premise-scm).
+- You must add or onboard a Project (repository) to SCP and it must complete at least one full scan successfully.
```

Comment:
Why is project capitalized?

I also ran into a wording issue with the full scan thing; the way this is currently worded makes it seem like the project has to complete the full scan -- maybe something like "you must have scanned it fully at least once with Semgrep?"

Reply:
1. Project is capitalized so they know it's a (Semgrep) Project - do you think we should continue this convention? (Happy to open a slack thread if necessary, otherwise we can just decide here).
2. Semgrep performs 2 kinds of scans, a "full scan" vs a diff-aware one. Both scans can actually be performed on unintended targets (full scan on PRs, diff-aware on main, we don't want people to do this, but it can happen.) We could also say:
- **Full scan on your default branch**
- Maybe: "your initial scan", because initial scans after adding to a project are full scans.
- 
What do you think?

Reply:
Not to go off the other deep end, but let's stick with Semgrep Project so that it is crystal clear?

I think both "full scan on your default branch" or "your initial scan" works -- I would lean toward the former, just in case someone is setting this up after they've done multiple scans

Reply:
Resolved based on convo in slack!